playbooks/roles/backups/defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 
 BACKUPS_MYSQL: yes
 BACKUPS_MONGO: yes
+BACKUPS_MONGO_USER: 'admin'
+BACKUPS_MONGO_PASSWORD: 'password'
```

Comment:
@johnbaldwin This is the default password and is overridden in customer server vars.